src/game_server/player_update_packet.rs

```diff
@@ -21,6 +36,215 @@ impl SerializePacket for PlayerUpdateOpCode {
     }
 }
 
+#[derive(SerializePacket, DeserializePacket)]
+pub struct LootEvent {
+	guid: u64,
+	position: Pos,
+	rotation: Pos,
```

Comment:
```suggestion
	pos: Pos,
	rot: Pos,
```
nit: pos and rot are more consistent with the names used in other packets